Cargo.toml

```diff
@@ -33,6 +33,7 @@ path = "tests/tests.rs"
 atty = "0.2.2"
 bytecount = "0.1.4"
 clap = "2.26"
+clippy = {version = "*", optional = true}
```

Comment:
Sorry, but please remove this and any corresponding `cfg`s. I don't want to add a very unstable linter to ripgrep's dependencies.

Reply:
This is not a default dependency, clippy is *only* pulled in if we explicitly build with "--features clippy"

Reply:
Yes, I understand. I don't want it.

Reply:
Alright, I removed it.